.github/workflows/build-test.yml

```diff
@@ -51,3 +51,10 @@ jobs:
           version: "2021.1.1"
           install-go: false
           cache-key: ${{ matrix.go }}
+
+      - name: Generate Cucumber Godog Test File
```

Comment:
I wonder if it makes sense to remove the `gremlin.go` file from revision control and just generate it before running docker-compose?